docs/modules/ROOT/pages/erc721.adoc

```diff
@@ -57,3 +57,19 @@ For more information about erc721 schema, check out the https://eips.ethereum.or
 
 NOTE: You'll notice that the item's information is included in the metadata, but that information isn't on-chain!
 So a game developer could change the underlying metadata, changing the rules of the game!
+
+[[erc721-token-extensions]]
+== Extensions
+Additionally, there are multiple custom extensions, including:
+
+* xref:erc721-burnable.adoc[ERC-721 Burnable]: destruction of own tokens.
+
+* xref:erc721-metadata.adoc[ERC-721 Metadata]: the extended ERC721 interface including the name, symbol, and decimals functions.
+
+* xref:erc721-pausable.adoc[ERC-721 Pausable]: ability to pause token transfers.
+
+* xref:erc721-consecutive.adoc[ERC-721 Consecutive]: ability to mint tokens consecutively.
+
+* xref:erc721-enumerable.adoc[ERC-721 Enumerable]: ability to iterate over all tokens.
```

Comment:
```suggestion
* xref:erc721-enumerable.adoc[ERC-721 Enumerable]: Optional extension that allows enumerating the tokens on chain, often not included since it requires large gas overhead.
```

Reply:
From here -> https://docs.openzeppelin.com/contracts/5.x/api/token/erc721